client/Models/EditorModel.ts

```diff
@@ -0,0 +1,17 @@
+import { atom } from 'jotai';
+
+// Default Editor Configurations
+export const editorSettingsAtom = atom({
+    theme: 'monokai',
+    options: [
+        'enable-auto-indent',
+        'show-line-numbers',
+        'enable-live-autocompletion',
+        'enable-live-autocompletion',
+    ],
```

Comment:
As we discussed, this array of strings should just be inlined fields that are booleans:
```suggestion
    enableAutoIndent: true,
    showLineNumbers: true,
    enableLiveAutocompletion: true,
```
(Don't accept this suggestion through GitHub)